.eslintrc.json

```diff
@@ -163,6 +163,19 @@
     "jsx-a11y/no-noninteractive-element-interactions": 1,
     "jsx-a11y/no-noninteractive-tabindex": 1,
     "jsx-a11y/no-redundant-roles": 1,
-    "jsx-a11y/role-has-required-aria-props": 1
+    "jsx-a11y/role-has-required-aria-props": 1,
+
+    // Taken from eslint-config-sfmc
+    "func-names": 0,
+    "import/first": 0,
+    "react/no-multi-comp": 0,
+
+
+    // When someone is ready to tackle these issues, feel free to uncomment the following
+    // Do not push a change that causes lint errors/warnings to start appearing
+    // All lint warnings errors should be meaningful, and therefore, there should never intentionally be any
+    "react/prefer-es6-class": 0,
```

Comment:
Did you just tackle this? sort comp too?

Reply:
No, these three settings can not be turned on without a substantial amount of re-work.